airflow/providers/google/cloud/operators/bigquery.py

```diff
@@ -467,6 +467,14 @@ def execute(self, context: 'Context') -> list:
             impersonation_chain=self.impersonation_chain,
         )
 
+        if not self.selected_fields:
+            schema: Dict[str, list] = hook.get_schema(
+                dataset_id=self.dataset_id,
+                table_id=self.table_id,
+            )
+            if schema.get("fields"):
+                self.selected_fields = ','.join([field["name"] for field in schema["fields"]])
```

Comment:
When is `"fields"` not present in `schema`, and what would happen?

Reply:
Here, `fields` are columns of a bigquery table so if no "fields" means there is no columns in bigquery table in that case we are passing `self.selected_fields` as  None in google client SDK and I'm getting an empty response so that won't be a problem.

Reply:
This should be `if "fields" in schema` then

Reply:
Make sense. Updated PR with the suggested change.